cosy.py

```diff
@@ -303,6 +303,11 @@ def check_completeness(symbols):
 
 
 if __name__ == "__main__":
+
+    # change to directory of cosy.py
+    dname = path.dirname(path.abspath(__file__))
+    chdir(dname)
```

Comment:
It is not needed to change directory, just keep the dname variable (maybe use a better name like `current_dir`) and use `os.path.join(dname, "root/symbols.json"` where needed.